src/components/Search/SearchTitle.css

```diff
@@ -0,0 +1,61 @@
+.SearchTitle {
+  padding-bottom: 1rem;
+  display: none;
+}
+
+.SearchTitle .SearchTitle__Container {
+  display: flex;
+  align-items: center;
+}
+
+.SearchTitle .dcl.back {
+  top: 1.2em;
+  left: -4em;
+  background-position-x: 10px;
+  background-position-y: 9px;
+}
+
+.BigBack {
+  min-width: 32px;
+  min-height: 32px;
+}
+
+.SmallBack {
+  min-width: 26px;
+  min-height: 26px;
+}
+
+.ui.header.SearchTitle__Text {
+  display: flex;
+  margin: 0;
+  padding-left: 24px;
+}
+
+.SearchTitle__Separator {
+  width: 100%;
+  border-color: #e1e0e3;
+  opacity: 0.2;
+  margin-top: 16px;
+}
+
+@media (max-width: 767px) {
+  .SearchTitle.OnlyMobile {
```

Comment:
Is `OnlyMobile` and `OnlyDesktop` classes necessary? You can handle this with media queries. Using this media query is "only mobile". I'd use a "mobile first" approach to avoid using max-width.